src/ci/resources/centaur_secure.inc.conf.ctmpl

```diff
@@ -1,13 +1,6 @@
-{{with $centaurSentry := vault (printf "secret/dsde/cromwell/common/centaur-sentry")}}
+# NOTE: This file will only be included if secrets can be rendered.
```

Comment:
I don't understand this statement. This file no longer contains any secrets as far as I can tell?

Reply:
This means there is nothing blocking someone from running local/tes/slurm/etc. centaur tests, except an ample amount of patience. This file doesn't render from `.conf.ctmpl` to `.conf`, thus the `.conf` isn't found and is just skipped, BigQuery secrets are not required, and therefore external users can still run tests.

The file itself doesn't contain secrets, but the stanza below refers to our google account. It says "if there's an error, then upload a report to DSP-Batch's BigQuery".

If one tries to put this BigQuery reference directly into the centaur.conf, then no one outside of Broad can run the tests, as they will not have access our google account to upload failure reports.